Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java

```diff
@@ -228,6 +213,12 @@ protected void onInvalidate() {
         mVungleRouterListener = null;
     }
 
+    @Nullable
+    @Override
+    protected LifecycleListener getLifecycleListener() {
+        return null;
```

Comment:
this might be good to evaluate later and add some implementation , for now think MoPub intentionally left as default null

Reply:
yes, something like  MoPub.onPause(getActivity());  called from developer side won't work
but at least we are handling this with View#onVisibilityChanged